net/WebSocketHandler.hpp

```diff
@@ -241,21 +242,52 @@ class WebSocketHandler : public ProtocolHandlerInterface
     void shutdown(const StatusCodes statusCode = StatusCodes::NORMAL_CLOSE,
                   const std::string& statusMessage = std::string(),
                   bool hardShutdown = false)
+    {
+        shutdownImpl(statusCode, statusMessage, hardShutdown, false);
+    }
+
+    /// Don't wait for the remote Websocket to handshake with us; go down fast.
+    void shutdownAfterWriting()
```

Comment:
Perhaps `shutdownFast`, `shutdownImmediately`, or `shutdownWithoutWaiting` or something more similar to the comment? 

What has "writing" anything to do with "Don't wait for the remove Websocket" and "go down fast" ? Seems misleading name.

Also, I expect it force-flushes before closing. In that case, it could be called `flushAndShutdown`, or some such.

Reply:
I just moved `shutdownAfterWriting` method and left the name unchanged.